ecdsa.go

```diff
@@ -322,17 +390,18 @@ func (a *ecdsaAlgo) decodePublicKeyCompressed(pkBytes []byte) (PublicKey, error)
 	if a.curve == elliptic.P256() {
 		x, y := elliptic.UnmarshalCompressed(a.curve, pkBytes)
 		if x == nil {
-			return nil, invalidInputsErrorf("Key %x can't be interpreted as %v", pkBytes, a.algo.String())
+			return nil, invalidInputsErrorf("input %x isn't a compressed deserialization of a %v key", pkBytes, a.algo.String())
 		}
 		goPubKey = new(ecdsa.PublicKey)
 		goPubKey.Curve = a.curve
 		goPubKey.X = x
 		goPubKey.Y = y
 
 	} else if a.curve == btcec.S256() {
+		// use `btcec` because elliptic's `UnmarshalCompressed` doesn't work for SEC Koblitz curves
 		pk, err := btcec.ParsePubKey(pkBytes)
 		if err != nil {
-			return nil, invalidInputsErrorf("Key %x can't be interpreted as %v", pkBytes, a.algo.String())
+			return nil, invalidInputsErrorf("input %x isn't a compressed deserialization of a %v key", pkBytes, a.algo.String())
```

Comment:
```suggestion
			return nil, invalidInputsErrorf("input %x isn't a compressed serialization of a %v key", pkBytes, a.algo.String())
```
nit: I would say the input is a serialization and this function deserializes it

Reply:
Agreed, this was a typo